openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -0,0 +1,57 @@
+export default function () {
+
+    function val(selector) {
+        return $.trim($(selector).val());
+    }
+    function error(message, event) {
+        $('#errors').show().html(message);
+        event.preventDefault();
+    }
+
+    // page may not be loaded via iframe
+    if (parent && parent.closeThrobber) {
+        parent.closeThrobber();
+    }
+    $('#form.addcover-form').on('submit', function(event) {
+        var file = val('#coverFile');
+        var url = val('#imageUrl');
+        var coverid = val('#coverid');
+
+        if (file === '' && (url === '' || url === 'http://') && coverid === '') {
+            return error('Please choose an image or provide a URL.', event);
+        }
+
+        function test_url(url) {
+            var obj = {
+                optional: function() { return false; }
+            }
+            return $.validator.url.apply(obj, [url, null]);
+        }
+
+        if (url != '' && url != 'http://' && !test_url(url)) {
```

Comment:
Again not introduced by you.. but why we're here should be double equals e.g. !==